schema/descriptor.schema.json

```diff
@@ -1,501 +0,0 @@
-{
```

Comment:
If it doesn't harm the Python package, I would make the symbolic link in the other direction (file in `schema`, link in `tools/python`) just in case other languages need to refer to it.